src/actions/updateJobs.js

```diff
@@ -3,6 +3,7 @@
 const Archetype = require('archetype');
 const assert = require('assert');
 const mongoose = require('mongoose');
+const {createMessage} = require('../integrations/slack');
```

Comment:
Better to do `const slack = require('../integrations/slack')` for sinon stubs. If you do `const {createMessage} =`, no way for tests to stub out `createMessage()`.